test/util-test.js

```diff
@@ -33,3 +33,42 @@ describe('base64encode', function() {
     assert.equal(utils.base64encode('\x01\x02\x99'), 'AQLCmQ==');
   });
 });
+
+describe('getMissingResources', function() {
+  it('returns an empty list given no response', function() {
+    assert.deepEqual(utils.getMissingResources(undefined), []);
+  });
+
+  it('returns an empty list given no response body', function() {
+    assert.deepEqual(utils.getMissingResources({}), []);
+  });
+
+  it('returns an empty list given no response body data', function() {
+    assert.deepEqual(utils.getMissingResources({ body: {} }), []);
+  });
+
+  it('returns an empty list given no response body data relationship', function() {
+    assert.deepEqual(utils.getMissingResources({ body: { data: {} } }), []);
+  });
+
+  it('returns an empty list given no missing resources', function() {
+    assert.deepEqual(
+      utils.getMissingResources({ body: { data: { relationships: {} } } }),
+      []
+    );
+  });
+
+  it('returns an empty list given no missing resources data', function() {
+    assert.deepEqual(
+      utils.getMissingResources({ body: { data: { relationships: { 'missing-resources': {} } } } }),
+      []
+    );
+  });
+
+  it('returns the missing resources', function() {
+    assert.deepEqual(
+      utils.getMissingResources({ body: { data: { relationships: { 'missing-resources': { data: [{ id: '123' }, { id: '456' }] } } } } }),
```

Comment:
Needs some formatting! :) We should probably prettyify this repo — maybe later.

Reply:
Haha yeah, get Prettier or at least ESLint on this repo if you want to enforce any styling conventions!